gpApi/index.js

```diff
@@ -104,22 +103,22 @@ const gpApi = {
       withAuth: true,
     },
     createDemoCampaign: {
-      url: `${base}campaign/demo`,
+      url: `${base}campaign/demo`, // TODO: remove? (not migrated)
       method: 'POST',
       withAuth: true,
     },
     deleteDemoCampaign: {
-      url: `${base}campaign/demo`,
+      url: `${base}campaign/demo`, // TODO: remove? (not migrated)
       method: 'DELETE',
       withAuth: true,
     },
```

Comment:
We will delete these.. maybe add this url to the ticket in the TODOs here?

https://goodparty.atlassian.net/browse/WEB-3583